.env

```diff
@@ -1 +1,23 @@
-REACT_APP_API_PROFILE = https://profile.pancakeswap.com
\ No newline at end of file
+# REACT_APP_CHAIN_ID = "56"
+REACT_APP_GTAG = "GTM-PXLD3XW"
+
+# 10+ nodes balanced, US/EU
```

Comment:
can you clean this ?

Reply:
Fixed in 4d61a939a5b316a75796bf5252f16bdd98f5285c